### ISSUE TITLE
windows freezes 

### ISSUE THREAD

**Issue by zaazii** [s0md3v/roop#51]
After I click on start, some processing takes place but then everything freezes and nothing solves it but manually restarting the computer. After that I find that the program created a folder with the extracted frames in it. No output file. This happened several times.

**Comment by s0md3v** [comment]
what OS? how much RAM? which CPU?

**Comment by zaazii** [comment]
> what OS? how much RAM? which CPU?

Windows 10
16 gb Ram
CPU	Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz

**Comment by zaazii** [comment]
In the installation there were two components that failed to install. But I ran the program anyway to see if it works. After restarting the computer I installed the program in a different folder but it didn't say anything about missing components. The program froze like the first time.

**Comment by s0md3v** [comment]
@zaazii which components failed to install?

**Comment by zaazii** [comment]
I didn't pay much attention. My intention was to first try the program, if it fails then I will get back to the message and install these components. But then everything froze and I lost the message. I am thinking of uninstalling and reinstalling cuda then try installing this program again.

**Comment by s0md3v** [comment]
Alright mate, keep us updated!

**Comment by zaazii** [comment]
> Alright mate, keep us updated!

I reinstalled Cuda and here's the error I am getting:
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
gensim 4.3.0 requires FuzzyTM>=0.4.0, which is not installed.
numba 0.56.4 requires numpy<1.24,>=1.18, but you have numpy 1.24.3 which is incompatible.

**Comment by s0md3v** [comment]
- install FuzzyTM.
- uninstall numpy and then install numpy 1.24.0

**Comment by zaazii** [comment]
> * install FuzzyTM.
> * uninstall numpy and then install numpy 1.24.0

I did that. But Windows froze.

**Comment by zaazii** [comment]
And after I installed numpy I got the following error:
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
seaborn 0.12.2 requires numpy!=1.24.0,>=1.17, but you have numpy 1.24.0 which is incompatible.
numba 0.56.4 requires numpy<1.24,>=1.18, but you have numpy 1.24.0 which is incompatible.
